tests/lib/systems.sh

```diff
@@ -14,16 +14,55 @@ is_core18_system(){
     return 1
 }
 
+is_core20_system(){
+    if [[ "$SPREAD_SYSTEM" == ubuntu-core-20-* ]]; then
+        return 0
+    fi
+    return 1
+}
+
 is_classic_system(){
     if [[ "$SPREAD_SYSTEM" != ubuntu-core-* ]]; then
         return 0
     fi
     return 1
 }
 
+
 is_ubuntu_14_system(){
     if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
         return 0
     fi
     return 1
 }
+
+get_snap_for_system(){
+    snap=$1
```

Comment:
If this is bash then we should be more careful with variables and use `local`. Alternatively, if you can, avoid the variables altogether, when it is natural to do so.

Reply:
Sure, thanks for reviewing this.